src/entities/raw-data-providers/cardano-brdige-api.js

```diff
@@ -66,8 +66,8 @@ class CardanoBridgeApi implements RawDataProvider {
     return resp
   }
 
-  async postSignedTx(payload: string) {
-    const resp = await this.post('txs/signed', payload)
+  async postSignedTx(signedTx: string) {
```

Comment:
I don't get it. It was already a string, why are we stringifying it? This is in byron, too, so I don't understand why we're changing things.